networkit/cpp/generators/test/GeneratorsBenchmark.cpp

```diff
@@ -38,15 +48,12 @@ TEST_F(GeneratorsBenchmark, benchmarkGraphBuilder) {
 
 	// prepare a random generator for each possible thread
 	int maxThreads = omp_get_max_threads();
-	std::vector< std::function<double()> > randomPerThread;
```

Comment:
I assume that the reason why we do not use NetworKit's PRNG is that we want this particular distribution?

Reply:
This point is not important anymore, since the code was removed a couple of commits later. (nevertheless, I think it's okay to not rebase it, since it documents the development process. This is one of those cases, where in the future we should have a clear Guide on how to do it.) The reason I did not use `Random::probability` at that point, was that I was not yet clear on the performance impact (which is now measured and document in  69dcccc).